vignettes/chapter0_Installation.Rmd

```diff
@@ -30,7 +30,7 @@ knitr::opts_chunk$set(echo = TRUE)
 ## Install GGIR
 
 Install the latest released version of GGIR with its dependencies from
-CRAN. You can do this with one command from the console command line:
+CRAN. You can do this with one command in your RStudio console:
```

Comment:
Not everyone who uses GGIR uses RStudio, because on clusters we usually use R directly. Maybe R command line is the best way of phrasing it.

Reply:
Agree, will revise this.